scripts/README.md

```diff
@@ -0,0 +1,217 @@
+
+### [decrypt_gha_secret.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/decrypt_gha_secret.sh)
+
+CI script to decrypt a GitHub Actions secret.
+
+### [encrypt_gha_secret.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/encrypt_gha_secret.sh)
+
+CI script to encrypt a GitHub Actions secret.
+
+### [fuzzing_ci.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/fuzzing_ci.sh)
+
+Firestore CI script to run fuzz testing.
+
+### [generate_access_token.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/generate_access_token.sh)
+
+Script to generate a Firebase access token used by Remote config integration tests.
+
+### [install_prereqs.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scriptsinstall_prereqs.sh)
+
+Utility CI script to provide configuration for build.sh
+
+### [localize_podfile.swift](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/localize_podfile.swift)
+
+Utility script to update a Podfile to point to local podspecs.
+
+### [make_release_notes.py](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/make_release_notes.py)
+
+Converts GitHub flavored markdown changelogs to release notes.
+
+### [pod_lib_lint.rb](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/pod_lib_lint.rb)
+
+Wrapper script for running `pod lib lint` tests to include dependencies from the monorepo.
+
+### [release_testing_setup.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/release_testing_setup.sh)
+
+Utility script for the release workflow.
+
+### [remove_data.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/remove_data.sh)
+
+Cleanup script for CI workflows.
+
+### [run_database_emulator.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/run_database_emulator.sh)
+
+Run the RTDB emulator.
+
+### [run_firestore_emulator.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/run_firestore_emulator.sh)
+
+Run the Firestore emulator.
+
+### [setup_bundler.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/setup_bundler.sh)
+
+Set up the Ruby bundler.
+
+### [setup_check.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/setup_check.sh)
+
+Install tooling for the check workflow.
+
+### [setup_quickstart.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/setup_quickstart.sh)
+
+Set up a QuickStart for integration testing.
+
+### [setup_quickstart_framework.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/setup_quickstart_framework.sh)
+
+Set up a QuickStart for zip distribution testing.
+
+### [setup_spm_tests.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/setup_spm_tests.sh)
+
+Configuration for SPM testing.
+
+### [spm_test_schemes](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/spm_test_schemes)
```

Comment:
Nit: maybe add a `/` after to indicate it's a directory? At first glance I was confused why we had a binary in here

Reply:
Done.